tools/RAiDER/utilFcns.py

```diff
@@ -100,51 +98,47 @@ def ecef2enu(xyz, lat, lon, height):
     return np.stack((e, n, u), axis=-1)
 
 
-def gdal_extents(fname):
+def rio_extents(fname):
     if os.path.exists(fname + '.vrt'):
         fname = fname + '.vrt'
-    try:
-        ds = gdal.Open(fname, gdal.GA_ReadOnly)
-    except Exception:
-        raise OSError('File {} could not be opened'.format(fname))
 
-    # Check whether the file is georeferenced
-    proj = ds.GetProjection()
-    gt = ds.GetGeoTransform()
+    with rasterio.open(fname) as src:
+        gt = src.transform.to_gdal()
+        proj = src.crs
+        xSize = src.width
+        ySize = src.height
+
     if not proj or not gt:
         raise AttributeError('File {} does not contain geotransform information'.format(fname))
```

Comment:
This can be simplified to be:

```
with rasterio.open(fname) as ds:
    bounds = ds.bounds

return ds.bounds
```

Reply:
+1 on this

Reply:
`bounds` appears to use a different convention from `SNWE` . Will keep it as is for now - unless we want to go in and update all usages of bbox.